compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -50,7 +50,8 @@ darPackageIds fp = do
     Right dalfPkgIds  <- pure $ mapM (LFArchive.decodeArchivePackageId . BSL.toStrict) $ mainDalf : dalfDeps
     pure dalfPkgIds
 
-
+-- TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
hopefully the test should tell us that by failing but I guess the todo doesn’t do any harm.

Reply:
I think we should probably refactor all of this (in a separate PR).
"numStablePackages" should be derived from a list of stable packages.